micromagneticdata/drive.py

```diff
@@ -418,26 +418,35 @@ def slider(self, description="step", **kwargs):
     def to_xarray(self, *args, **kwargs):
         """Export ``micromagneticdata.Drive`` as ``xarray.DataArray``
 
-        The method depends on ``discretisedfield.Field.to_xarray()`` and derives the
+        The method depends on ``discretisedfield.Field.to_xarray`` and derives the
         last four dimensions ``x``, ``y``, ``z``, and ``comp`` in the output
-        ``xarray.DataArray`` from it.
-
-        Depending on ``micromagneticdata.Drive.info['driver']``, the dimensions and
-        coordinates of the output changes. For ``MinDriver``, which only contains a
-        single ``discretisedfield.Field``, the value of the single
-        ``discretisedfield.Field.to_xarray()`` is returned. For ``TimeDriver``, the
-        output contains an extra dimension ``t`` which corresponds to the time steps of
-        the simulation and is obtained by concatenating individual
-        ``discretisedfield.Field.to_xarray()`` in the ``micromagneticdata.Drive``. The
-        coordinates of ``t`` give the values of time at given time steps. Similarly,
-        for ``HysteresisDriver``, the extra dimension is ``B_hysteresis``. It has four
-        coordinates, namely ``B_hysteresis``, ``Bx_hysteresis``, ``By_hysteresis``, and
+        ``xarray.DataArray`` from it. The arguments and named arguments to this method
+        are passed on to ``discretisedfield.Field.to_xarray``.
+
+        Depending on type of driver, the dimensions and coordinates of the output may
+        change. If the number of stored steps in the ``micromagneticdata.Drive`` are
+        more than one, the output contains an extra dimension named after
+        ``micromagneticdata.Drive.table.x`` with proper coordinate values. For the case
+        of ``HysteresisDriver``,  the new dimension has four coordinates, namely
+        ``B_hysteresis``, ``Bx_hysteresis``, ``By_hysteresis``, and
         ``Bz_hysteresis``. The first represents the norm of the hysteresis field, while
-        the rest three represents the components along the respective axes.
+        the rest three represents the components along the respective axes. For
+        ``MinDriver`` with a single ``discretisedfield.Field``, the value of the single
+        ``discretisedfield.Field.to_xarray`` is returned.
```

Comment:
Have you tried a time drive with a single step?

Reply:
The method will return the `discretisedfield.Field.to_xarray` in that case as well. I have updated the docstrings accordingly.